src/main/java/core/basesyntax/dao/FileReaderImpl.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.dao;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+public class FileReaderImpl implements FileReader {
+
+    @Override
+    public String[] read(String nameOfFile) {
+        List<String> textInFile = null;
+        try (BufferedReader reader = new BufferedReader(new java.io.FileReader(nameOfFile))) {
```

Comment:
import FileReader, don't write fully qualified name

Reply:
Not fixed, maybe will be better to change the name of your `FileReader, FileWriter` interfaces to something like `CsvFileReader, CsvFileWriter`. It will help you to avoid fully qualified name `java.io.FileReader`